CLAMSSpeciesFix.py

```diff
@@ -85,14 +89,14 @@ def __init__(self, parent=None):
         self.setMaximumSize(window.width(), window.height())
 
         # general slots
-        self.connect(self.doneBtn, SIGNAL("clicked()"), self.goExit)
-        self.connect(self.startIDBtn, SIGNAL("clicked()"), self.getIDRange)
-        self.connect(self.clearBtn, SIGNAL("clicked()"), self.clearFilters)
-        self.connect(self.endIDBtn, SIGNAL("clicked()"), self.getIDRange)
-        self.connect(self.applyChangeBtn, SIGNAL("clicked()"), self.applyChange)
-        self.connect(self.scientistBox, SIGNAL("activated(int)"), self.filterMeasurements)
-        self.connect(self.workstationBox, SIGNAL("activated(int)"), self.filterMeasurements)
-        self.connect(self.speciesBox, SIGNAL("activated(int)"), self.filterMeasurements)
+        self.doneBtn.clicked.connect(self.goExit)
+        self.startIDBtn.clicked.connect(self.getIDRange)
+        self.clearBtn.clicked.connect(self.clearFilters)
+        self.endIDBtn.clicked.connect(self.getIDRange)
+        self.applyChangeBtn.clicked.connect(self.applyChange)
+        self.scientistBox.clicked.connect(self.filterMeasurements)
```

Comment:
These signal names are not correct. Use the signal name that is defined in the `SIGNAL` function in the old code. Lines 92-96 are correct since those are push buttons that emit the `clicked` signal when depressed, but lines 97-99 are not correct since those objects are combo boxes and we're  connecting to the `activated` signal. Specifically, the activated signal that returns an integer. The correct form would be:

`self.scientistBox.activated[int].connect(self.filterMeasurements)`